confidential-data-hub/hub/src/config.rs

```diff
@@ -227,6 +326,7 @@ path = "/run/confidential-containers/cdh/kms-credential/aliyun/config.toml"
             },
             credentials: Vec::new(),
             socket: DEFAULT_CDH_SOCKET_ADDR.into(),
+            image: crate::ImageConfiguration::default(),
```

Comment:
Sorry if I've misunderstood something, but is it worth the unit test config file (or another test) having non-default values, so we can check the file reading is working and not just failing, but falling back on Default?

Reply:
Nice suggestion. Let me add a simple unit test w/o any field and see if all will be filled with default values. Also another test with an undefined field, and see if the config parser would ignore that field and fill other fields w/ default values.